### PR TITLE
[Polymetis] EE tracking benchmark (EE tracking improvement #1)

### PR DESCRIPTION
**Pull request by exhaustin**
# Description

Add script for benchmarking EE tracking performance by drawing a Meta logo in 3D space.
Any future change that affects EE control performance can be evaluated using this script.

### End-effector trajectory visualization
Screen Shot 2022-08-09 at 11 05 36 AM
image
image

## Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] Proposes a change (non-breaking change that isn't necessarily a bug)
- [ ] Refactor
- [x] New feature (non-breaking change that adds a new functionality)
- [ ] Breaking change (fix or feature that would break some existing functionality downstream)
- [ ] This is a unit test
- [ ] Documentation only change
- [ ] Datasets Release
- [ ] Models Release

## Type of requested review

- [x] I want a thorough review of the implementation.
- [ ] I want a high level review. 
- [ ] I want a deep design review.

## Before and After

If this PR introduces a change or fixes a bug, please add before and after screenshots (if this is causes a UX change) or before and after examples(this could be plain text, videos etc ) to demonstrate the change.

If this PR introduces a new feature, please also add examples or screenshots demonstrating the feature.

# Testing

Results of running the benchmark with the current version of Polymetis:
```
=== Offline (move_to_joint_positions) tracking results ===
        Pos error(mm): mean=10.92, std=7.60, max=34.40
        Ori error(deg): mean=2.69, std=0.71, max=4.78
=== Offline (move_to_ee_pose) tracking results ===
        Pos error(mm): mean=43.66, std=16.01, max=74.56
        Ori error(deg): mean=6.76, std=1.61, max=8.76
=== Online (update_desired_ee_pose) tracking results ===
        Pos error(mm): mean=37.78, std=10.35, max=56.27
        Ori error(deg): mean=11.31, std=2.64, max=15.30
```

# Checklist:

- [x] I have performed manual end-to-end testing of the feature in my environment.
- [ ] I have added Docstrings and comments to the code.
- [ ] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have added relevant collaborators to review the PR before merge.
- [ ] [Polymetis only] I ran on hardware (1) all scripts in `tests/scripts`, (2) asv benchmarks.

### CODE REVIEW

**Review by 0wu — APPROVED** [review]
LGTM

File: polymetis/polymetis/python/scripts/benchmark_ee_tracking.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+        dz = radius * np.sin(theta * 2)
+        ee_pose_traj[i, :3] = center_pose[:3] + torch.Tensor([dx, dy, dz])
+
+        # Orientation: Stay in place
+        ee_pose_traj[i, 3:] = center_pose[3:]
+
+    return ee_pose_traj
+
+
+def compare_traj(ref_traj, states_reached, robot_model, experiment_name=""):
```
Comment (1heart) [inline polymetis/polymetis/python/scripts/benchmark_ee_tracking.py:47]:
```suggestion
@torch.no_grad()
def compare_traj(ref_traj, states_reached, robot_model, experiment_name=""):
```

**Review by 1heart — APPROVED** [review]
Two improvements, which could be in future PRs:

1. Visualization of the trajectory in Cartesian space
2. Adding it to the set of hardware benchmarks that get tracked over time.

Otherwise, lgtm!

**Comment by exhaustin** [comment]
> Two improvements, which could be in future PRs:
> 
> 1. Visualization of the trajectory in Cartesian space
> 2. Adding it to the set of hardware benchmarks that get tracked over time.
> 
> Otherwise, lgtm!

Moved the EE benchmark script to `tests/scripts` and added visualizations to the PR description!

**Pull request merged**